### ISSUE TITLE
Can't detect Logitech trackpad

### ISSUE THREAD

**Issue by pilotmoon** [pilotmoon/Scroll-Reverser#4]
Ref: **[Conversation #415](https://secure.helpscout.net/conversation/4158830/415) / Keith**

Hi,

I'm using my Macbook Pro Retina at a desk with both a mouse and one of the relatively recent Logitech Trackpads for Mac[1](http://www.logitech.com/en-us/product/mac-trackpad?crid=1153). I use Scroll Reverser set to just "Reverse Mouse".

Here's what I expect to happen with Scroll Reverser running:

Mouse wheel - scroll forward scrolls up
Built-in trackpad - two-finger swipe forward scrolls down
External trackpad - two-finger swipe forward scrolls down

But here's what actually happens:

Mouse wheel - scroll forward scrolls up
Built-in trackpad - two-finger swipe forward scrolls down
External trackpad - two-finger swipe forward scrolls UP

In other words, it seems Scroll Reverser thinks the external trackpad is actually a mouse and reverses it. Is there any way to work around this?

Thanks!

**Comment by mckennapsean** [comment]
I had the same issue with a Logitech T651 trackpad, getting scrolling inverted when only "Reverse Mouse" is checked.

A hacky fix for me was to use Logitech's driver program/settings and change the Scroll Direction to "Standard", rather than Natural. Then, when it gets inverted, it works as expected. I did have to invert both horizontal and vertical using Scroll-Reverser.
